mailpoet/lib/Services/AuthorizedSenderDomainController.php

```diff
@@ -235,9 +244,21 @@ private function returnAllDomains(array $records): array {
     return $domains;
   }
 
+  private function reloadCache() {
+    $this->currentRecords = null;
+    $this->currentRawData = $this->bridge->getRawSenderDomainData();
+    $this->wp->setTransient(self::SENDER_DOMAINS_KEY, $this->currentRawData, 60 * 60 * 24);
+  }
```

Comment:
Extremely minor comment since this looks fine as is.

This method looks slightly inconsistent. It's simply setting `currentRecords` to `null` and doesn't reload them. I think this might be simpler if instead it were called `clearCache`, and you could set `currentRawData` to `null` as well, and also delete the transient.

It would work exactly the same. The only real benefit would be not having to worry about updating the transient time in multiple places, but that could easily be fixed with a class constant... so again, no big deal.

Reply:
I wanted to avoid another API call. 
I am not fully sure when `$this->currentRecords` would need to be set to `null` or even if we still need it.
I didn't have time to look at it in detail but I can open a Janitorial ticket to do some cleanup here.

Reply:
I added some cleanup to the ticket to remove the enforcement date, including checking `$this->currentRecords` and the function `reloadCache`